Makefile

```diff
@@ -200,7 +200,7 @@ e2e-build-run-sh: build test-e2e-sh
 ################################################################################
 .PHONY: modtidy
 modtidy:
-	go mod tidy -compat=1.19
+	go mod tidy -compat=1.20
```

Comment:
1.20.1 ?

Reply:
`-compat` do not support patch version. Got the following if I try to put patch version there.

```
go mod tidy -compat=1.20.1
invalid value "1.20.1" for flag -compat: expecting a Go version like "1.20"
usage: go mod tidy [-e] [-v] [-x] [-go=version] [-compat=version]
Run 'go help mod tidy' for details.
make: *** [modtidy] Error 2

```